### PR TITLE
remove originality

### PR DESCRIPTION
**Pull request by kumikoda**
image

cc @kwgoodman @uuazed @BryantLuu

### CODE REVIEW

File: numerapi/numerapi.py [hunk 1]
```diff
@@ -1195,11 +1180,7 @@ def submission_status(self, submission_id=None):
         query = '''
             query($submission_id: String!) {
               submissions(id: $submission_id) {
-                originality {
-                  pending
-                  value
-                }
-                concordance {
+                oncordance {
```
Comment (uuazed) [inline numerapi/numerapi.py:1183]: the `c` is missing

File: tests/test_numerapi.py [hunk 2]
```diff
@@ -214,23 +214,20 @@ def test_check_new_round(api):
 
 
 @responses.activate
-@pytest.mark.parametrize('''originality_pending, originality_value,
-    concordance_pending, concordance_value, consistency, expected''', [
+@pytest.mark.parametrize('''concordance_pending, concordance_value,
+                            consistency, expected''', [
```
Comment (uuazed) [inline tests/test_numerapi.py:218]: this test will fail now

**Review by uuazed — COMMENTED** [review]
I'll merge your changes and fix the tests myself

File: tests/test_numerapi.py (hunk 2 above)
Comment (kumikoda) [reply tests/test_numerapi.py:218]: ah whoops sorry. thanks for fixing!

**Pull request merged**